project/maintainers.md

```diff
@@ -0,0 +1,221 @@
+<!-- vim: filetype=markdown colorcolumn=80
+-->
+# Maintainers handbook
+
+This document is a guide for maintainers across the spire/helm-chart effort to
+provide a consistent user interface for those going through the review process.
+
+## The Merge Request
+
+Merge requests are submitted through GitHub.  They are contributions to change
```

Comment:
In GitHub we call this `Pull Requests`. `Merge Requests` is terminology from GitLab.

Reply:
As long as it is consistent, I'm happy with this request.